src/redux/selectors/index.js

```diff
@@ -203,6 +203,41 @@ const filteredItemsInDateRangeSelector = createSelector(
   },
 );
 
+export const selectedRecordsGroupedByTypeSelector = createSelector(
+  [filteredItemsInDateRangeSelector, activeCollectionResourceTypeSelector],
+  (items, selectedResourceType) => {
+    const typeMap = items
+      .reduce((acc, item) => {
+        const { type, subType } = item;
+        return produce(acc, (draft) => {
+          // eslint-disable-next-line no-param-reassign
+          draft[type] = draft[type] ?? {};
+          // eslint-disable-next-line no-param-reassign
+          draft[type][subType] = draft[type][subType] ?? [];
+          draft[type][subType].push(item.id);
+        });
+      }, {});
+
+    const sortedGroupedResources = Object.entries(typeMap).reduce((acc, [type, subTypes]) => {
```

Comment:
minor, but: if  `.entries` and `.reduce` each start a new line, it will line up w/ the last`.sort`